superset/assets/javascripts/explorev2/components/controls/Filter.jsx

```diff
@@ -55,6 +58,15 @@ export default class Filter extends React.Component {
     if (event && event.value) {
```

Comment:
Gotcha. overall it would be more clear to have two functions, one for `changeOp` and one for `changeValue`